src/main/java/org/junitpioneer/jupiter/NotWorkingExtension.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2016-2021 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
+import org.junit.jupiter.api.extension.ExtensionContext.Store;
+import org.junit.jupiter.api.extension.LifecycleMethodExecutionExceptionHandler;
+import org.junit.jupiter.api.extension.TestExecutionExceptionHandler;
+import org.opentest4j.TestAbortedException;
+
+class NotWorkingExtension implements Extension, TestExecutionExceptionHandler, LifecycleMethodExecutionExceptionHandler,
+		BeforeEachCallback, AfterEachCallback {
+
+	private static final Namespace NAMESPACE = Namespace.create(NotWorkingExtension.class);
+	private static final String EXCEPTION_OCCURRED_STORE_KEY = "exceptionOccurred";
+
+	public NotWorkingExtension() {
+	}
+
+	private Store getStore(ExtensionContext context) {
```

Comment:
To be honest, I'd inline this method, but that's just my personal preference.

Reply:
If you don't mind I would like to keep the method for now to avoid having to repeat the `getStore(NAMESPACE)` part, which is a bit more verbose.